cli/tutors-json-manager/src/controller.ts

```diff
@@ -0,0 +1,77 @@
+import { encryptData, decryptData } from './encryption';
+import { compressData, decompressData } from './compression';
+import logger from './logger';
+
+interface EncryptedCompressedData {
+  isCompressed: boolean;
+  isEncrypted: boolean;
+  data: string;
```

Comment:
Structurally what I am aiming for here is two booleans to allow me know the order of operations as the encryption happens before the compression so the inverse has to occur in that order to successfully return the file